spec/system/feedback_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe 'User Feedback', type: :system do
+  before(:all) do
+    # These tests require the seed data to be present
+    Rails.application.load_seed
```

Comment:
Most other tests depend on a clean database (no seeds) so I'm loading seeds here. Loading seeds is also slow enough that we don't want to do it if we don't need to in a test run.

Reply:
This app is older than npm. There is probably a better way to get yarn on travis by now.